### PR TITLE
Add integration test

### PR DESCRIPTION
**Pull request by jhurliman**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ on:
     branches: [main]
 
 jobs:
-  build:
+  test:
```
Comment (defunctzombie) [inline .github/workflows/ci.yml:10]: Why rename this?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -35,7 +35,7 @@ jobs:
       - name: Configure Node.js
         uses: actions/setup-node@v2.1.5
         with:
-          node-version: ${{ matrix.node-version }}
+          node-version: 15.x
```
Comment (defunctzombie) [inline .github/workflows/ci.yml:38]: Fascinating - we didn't have a node-version in the matrix - what was this doing before?

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -90,4 +104,8 @@ jobs:
   #         restore-keys: |
   #           ${{ runner.os }}-yarn-
   #     - run: yarn install --immutable
-  #     - run: yarn run ${{ matrix.config.command }}
+  #     - name: test:integration
```
Comment (defunctzombie) [inline .github/workflows/ci.yml:107]: Lets remove the 3rd party action and put back `yarn run test:integration` along with changing the "runs-on" to `windows-latest`.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (jhurliman) [reply .github/workflows/ci.yml:10]: I was breaking build out to a separate top-level item that ran on Windows. If we end up running integration tests on Windows this isn't necessary.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (jhurliman) [reply .github/workflows/ci.yml:38]: :shrug:

**Pull request merged**